app/controllers/api/notes_controller.rb

```diff
@@ -81,19 +81,22 @@ def create
       # Extract the arguments
       lon = OSM.parse_float(params[:lon], OSM::APIBadUserInput, "lon was not a number")
       lat = OSM.parse_float(params[:lat], OSM::APIBadUserInput, "lat was not a number")
-      comment = params[:text]
+      description = params[:text]
+
+      # Get note's author info
+      note_author_info = author_info
 
       # Include in a transaction to ensure that there is always a note_comment for every note
       Note.transaction do
         # Create the note
-        @note = Note.create(:lat => lat, :lon => lon)
+        @note = Note.create(:lat => lat, :lon => lon, :description => description, :user_id => note_author_info[:user_id], :user_ip => note_author_info[:user_ip])
         raise OSM::APIBadUserInput, "The note is outside this world" unless @note.in_world?
 
         # Save the note
         @note.save!
 
-        # Add a comment to the note
-        add_comment(@note, comment, "opened")
+        # Add opening comment to the note
+        add_comment(@note, "", "opened")
```

Comment:
In PR we are getting rid of first comment as mechanism to store body and author info. I'm not sure should we also consider removing the first comment completly, eg. should the mechanism for opening/closing be refactored completly since we are getting rid of first comment logic. 

This could also be done in separate PR and maybe can be a part of conversation when PR is opened with the maintaners.